src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -439,6 +439,10 @@ def __init__(
         tokenizer: AutoTokenizer,
     ) -> None:
         self._tokenizer = tokenizer
+        # disable add_bos_token so that llama-tokenizer does not add bos token
+        # to the beginning of every response outputs, increasing the token count
+        # by twice.
```

Comment:
by 2x

Reply:
This increases token count by 1.  In the case where our chunks are 32 tokens, it would have less of an effect.

Reply:
I see. appreciate you explaining.